pkg/registry/input_stream.go

```diff
@@ -90,7 +90,7 @@ func (r *ReplacesInputStream) canAdd(bundle *Bundle, packageGraph *Package) erro
 
 	if replaces != "" && !packageGraph.HasCsv(replaces) {
 		// We can't add this until a replacement exists
-		return fmt.Errorf("Invalid bundle %s, bundle specifies a non-existent replacement %s", bundle.Name, replaces)
+		return fmt.Errorf("Invalid bundle %s, replaces nonexistent bundle %s", bundle.Name, replaces)
```

Comment:
This feels more like a personal choice and unrelated to the fix the PR is trying to address.

Reply:
sure, but the existing wording doesn't make sense.

Reply:
If we feel that was we can probably open another housekeeping PR that's titled `fix log message`. I'm just not sure if changing log message goes with `respect strict mode`

Reply:
I feel strongly that we shouldn't tax CI for trivial changes. I think the k8s docs also suggest that trival changes should go into larger PRs for that project.